src/levanter/config.py

```diff
@@ -93,26 +93,51 @@ def _maybe_get_config_path_and_cmdline_args(args: List[str]):
     If URL, we need to download it and save it to a temp file. We then want to remove --config_path
     from the cmdline args so that draccus doesn't try to load it as a config path and return it separately here
     along with the modified cmdline args.
+    We also accept ... --configs <config1> <config2> ... and concatenate them into a single config.
     """
-    if "--config_path" not in args and "--config" not in args:
+    if "--config_path" not in args and "--config" not in args and "--configs" not in args:
         return None, args
     else:
-        try:
-            config_path_index = args.index("--config_path")
-        except ValueError:
-            config_path_index = args.index("--config")
-
-        config_path = args[config_path_index + 1]
-
-        if urllib.parse.urlparse(config_path).scheme:
-            fs: AbstractFileSystem
-            fs, fs_path = fsspec.core.url_to_fs(config_path)
-            temp_file = tempfile.NamedTemporaryFile(prefix="config", suffix=".yaml", delete=False)
-            atexit.register(lambda: os.unlink(temp_file.name))
-            fs.get(fs_path, temp_file.name)
-            config_path = temp_file.name
+        config_args = ["--config_path", "--config", "--configs"]
+        found_indices = [args.index(arg) for arg in config_args if arg in args]
+        if len(found_indices) > 1:
+            raise ValueError(f"Multiple config args found in {args}")
+        config_path_index = found_indices[0] + 1
 
+        config_paths = []
         args = args.copy()
-        del args[config_path_index]
-        del args[config_path_index]
-        return config_path, args
+        del args[config_path_index - 1]
+        config_path_index -= 1
+
+        while config_path_index < len(args) and not args[config_path_index].startswith("-"):
+
+            config_path = args[config_path_index]
+
+            if urllib.parse.urlparse(config_path).scheme:
+                fs: AbstractFileSystem
+                fs, fs_path = fsspec.core.url_to_fs(config_path)
+                temp_file = tempfile.NamedTemporaryFile(prefix="config", suffix=".yaml", delete=False)
+                atexit.register(lambda: os.unlink(temp_file.name))
+                fs.get(fs_path, temp_file.name)
```

Comment:
I know this was there before, but can we just read the config instead of copying it into a temporary file, especially since we're going to be merging?  And if a config file has `foo: 3\nfoo:4`, does override work properly?

Reply:
we should just move all this logic into draccus

Reply:
Incase we want to read the config directly, we should probably do it in Draccus. Also override works properly right now.